### PR TITLE
Closes #265 | Implement dataloader for `myxnli`

### LINKED ISSUE #265: Create dataset loader for myXNLI
Dataloader name: `myxnli/myxnli.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?myxnli

|     Dataset| myxnli |
|-------------|---|
| Description | The myXNLI corpus is a collection of Myanmar language data designed for the Natural Language Inference (NLI) task, which originated from the XNLI and MultiNLI English datasets. The 7,500 sentence pairs from the XNLI English development and test sets are human-translated into Myanmar. The 392,702 data from the NLI English training data is translated using machine translation. In addition, it also extends its scope by adding Myanmar translations to the XNLI 15-language parallel corpus, to create a 16-language parallel corpus. |
| Subsets | - |
| Languages | mya |
| Tasks | Natural Language Inference |
| License | Creative Commons Attribution Non Commercial 4.0 (cc-by-nc-4.0) |
| Homepage | https://github.com/akhtet/myXNLI |
| HF URL | https://huggingface.co/datasets/akhtet/myXNLI |
| Paper URL | - |

### PR DESCRIPTION
**Pull request by chenxwh**
Closes #265.

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py`.
- [x] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: seacrowd/sea_datasets/myxnli/myxnli.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+class MyXNLI(datasets.GeneratorBasedBuilder):
+    """The myXNLI corpus is a collection of Myanmar language data designed for the Natural Language Inference task."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="myxnli_source",
```
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/myxnli/myxnli.py:54]: I suggest using `_DATASETNAME` here and in the `subset_id`.

File: seacrowd/sea_datasets/myxnli/myxnli.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+}
+
+_SUPPORTED_TASKS = [Tasks.TEXTUAL_ENTAILMENT]
+
+_SOURCE_VERSION = "1.1.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class MyXNLI(datasets.GeneratorBasedBuilder):
```
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/myxnli/myxnli.py:46]:
```suggestion
class MyXNLIDataset(datasets.GeneratorBasedBuilder):
```

File: seacrowd/sea_datasets/myxnli/myxnli.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        SEACrowdConfig(
+            name="myxnli_seacrowd_pairs",
+            version=SEACROWD_VERSION,
+            description="myxnli SEACrowd schema",
+            schema="seacrowd_pairs",
+            subset_id="myxnli",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "myxnli_source"
```
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/myxnli/myxnli.py:69]: Let's use `_DATASETNAME` here.

**Review by ljvmiranda921 — COMMENTED** [review]
Dataloader works as expected, but I have some small nits (below). I'm not sure if text pairs need a `meta` key, in my opinion. 

Also, you're missing an `__init__.py` file.

**Comment by chenxwh** [comment]
Thanks! Updated!

**Review by ljvmiranda921 — APPROVED** [review]
LGTM!

**Comment by danjohnvelasco** [comment]
The code is working as expected. Thank you for your PR!

**Pull request merged**